about.html

```diff
@@ -1,7 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
   <head> </head>
-  <body>
-    Hi I'm Andy.
+  <body style="font-family: monospace">
+    </p>Andy Nystrom is a husband and father to an 8 year old yellow lab mix named
+      Lucy.<br />
+      Previously he was a Director at Insound.com / Warner Music Group as well as
+      a Director of Audience Growth, Ad Operations, E-commerce, and Digital
+      Marketing at The FADER.<br />
+      Andy is mostly obsessed with music and vinyl records (all genres, mostly
+      1980 - today), alternative movie posters and pop art, comic books and
+      graphic novels, and coding.<br />
+      Originally from Long Island, NY, he now lives in Long Island City, NY.
+    </p>
+    <p>Email:
+    <a href="mailto: andrew.nystrom@gmail.com" target="_blank"
```

Comment:
Can you remove the space after the colon?

Reply:
thanks for catching, removed space.